index.js

```diff
@@ -24,9 +24,22 @@ const {
   RNReaderSDKStoreCustomerCard,
 } = NativeModules;
 
+function createReaderSDKError(ex) {
+  try {
+    const errorDetails = JSON.parse(ex.message);
+    ex.message = errorDetails.message; // eslint-disable-line no-param-reassign
+    ex.debugCode = errorDetails.debugCode; // eslint-disable-line no-param-reassign
+    ex.debugMessage = errorDetails.debugMessage; // eslint-disable-line no-param-reassign
+  } catch (parseEx) {
+    ex.parseEx = parseEx; // eslint-disable-line no-param-reassign
+  }
+  return ex;
+}
+
 export async function authorizeAsync(authCode) {
   try {
-    return await RNReaderSDKAuthorization.authorize(authCode);
+    const value = await RNReaderSDKAuthorization.authorize(authCode);
+    return value;
```

Comment:
Do we need assign a `value` before return?

Reply:
See you updated other places too, please let me know if that's necessary

Reply:
Without the changes, eslint gives an error stating: error  Returning an awaited value that is not a promise is not allowed  @typescript-eslint/return-await